exercises/lesson_03/README.rst

```diff
@@ -0,0 +1,69 @@
+==========
+Assignment
+==========
+
+In this assignment, you're going to practice locating files in your system as
+well as compression methods.
+
+1) Find all manual pages in your system which contain the string ``et`` in
+   their name (Note: manual pages are files located somewhere in your system).
+2) Copy the manual pages found in the previous step into the
+   ``mans_with_et_in_name`` directory in your home directory. Collect the names
+   of these manual pages, sort them alphabetically and write the resulting list
+   to the ``mans_with_et_in_name.txt`` file. The file contents should be
+   similar to this:
+
+   ::
+
+         $ head mans_with_et_in_name.txt
+         agetty.8.gz
+         charsets.7.gz
+         cpuset.7.gz
+         dmsetup.8.gz
+         dnssec-settime.8.gz
+         ethtool.8.gz
+         getcap.8.gz
+         getenforce.8.gz
+         getpcaps.8.gz
+
+3) Take the manual pages and decompress them.
+
+4) Once decompressed, look for files which contain any occurrence of the word
+   ``group`` (Note: look for whole words, not substrings). Take files which
+   returned a match on the search and copy them to the
+   ``mans_containing_word_group`` directory. Similarly to step 2) create a
+   sorted list of these files and save it in the
+   ``mans_containing_word_group.txt`` using the following format:
```

Comment:
You should specify where to place the .txt file. To home dir or inside created dir?